base/applications/mspaint/resource.h

```diff
@@ -165,6 +171,11 @@
 #define IDD_STRETCHSKEWEDITVSKEW    757
 #define IDD_STRETCHSKEWTEXTVDEG     758
 
+#define IDD_FONTS 760
```

Comment:
Please indent the values (and same below for IDS_BOLD and friends).

Reply:
Fixed.